widgets/opal/dialog/org.eclipse.nebula.widgets.opal.dialog/src/org/eclipse/nebula/widgets/opal/dialog/FooterArea.java

```diff
@@ -203,50 +222,66 @@ public void run() {
 	 * @param numberOfColumns
 	 */
 	private void createDetails(final int numberOfColumns) {
-		final Label icon = new Label(composite, SWT.NONE);
-		icon.setBackground(getGreyColor());
-		icon.setImage(isExpanded() ? getFewerDetailsImage() : getMoreDetailsImage());
-		icon.setLayoutData(new GridData(GridData.CENTER, GridData.CENTER, false, false));
-
-		final Label label = new Label(composite, SWT.NONE);
-		label.setBackground(getGreyColor());
-		label.setText(isExpanded() ? expandedLabelText : collapsedLabelText);
-		label.setLayoutData(new GridData(GridData.BEGINNING, GridData.CENTER, false, false));
+		ToolItem detailsItem = new ToolItem(toolbar, SWT.NONE);

```

Comment:
Please add final keyword before declaring the variable

Reply:
done